OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -520,7 +520,7 @@ public static DreamValue NativeProc_clamp(DreamObject instance, DreamObject usr,
             List<DreamValue> locs = arguments.GetAllArguments();
 
             foreach (DreamValue loc in locs) {
-                if (loc.Type != DreamValueType.DreamObject || loc.Value == null || !loc.GetValueAsDreamObject().IsSubtypeOf(DreamPath.Turf)) {
+                if (loc.Type != DreamValueType.DreamObject || loc.Value == null || !loc.TryGetValueAsDreamObjectOfType(DreamPath.Turf, out _)) {
```

Comment:
The other checks are already done by `TryGetValueAsDreamObjectOfType`
```suggestion
                if (!loc.TryGetValueAsDreamObjectOfType(DreamPath.Turf, out _)) {
```